package.json

```diff
@@ -12,7 +12,7 @@
   "repository": "https://github.com/atom/one-dark-ui",
   "main": "lib/main",
   "engines": {
-    "atom": ">0.40.0"
+    "atom": ">1.0.11"
```

Comment:
We strip everything after the patch version in a dev version like `1.0.12-dev-87cb24c`, right @atom/feedback? So this semver range should resolve as expected?